examples/handtrack/say_hello.html

```diff
@@ -11,8 +11,7 @@
             rel="stylesheet"
             href="https://pyscript.net/latest/pyscript.css"
         />
-        <script defer src="../../pyscriptjs/build/pyscript.js"></script>
-        <!-- <script defer src="https://pyscript.net/latest/pyscript.js"></script> -->
+        <script defer src="https://pyscript.net/latest/pyscript.js"></script>
```

Comment:
We're retiring `latest` right..? This should be a versioned link..?

Reply:
So, many examples have the same link i.e. `https://pyscript.net/latest/pyscript.js`

You can checkout the `antigravity.html` in the `examples` folder in the current state of the repository here: https://github.com/pyscript/pyscript/blob/main/examples/antigravity.html#L11

I guess we can address everything related to examples in a separate / follow-up PR?

The purpose of this specific change was to at least make things consistent, even if they are wrong and are to be cleaned up eventually...

Reply:
Ack. OK. We definitely need to get the examples story straight. For the record (and reflecting the conversation with @antocuni on Slack this morning):

* Examples will be removed from this repository.
* They will have their own versioned repository (tracking the PyScript version).
* The examples will be synced with versions on PyScript.com
* We're retiring `latest` and the examples will be pinned to specific versions of PyScript.

Right?

Reply:
yes, we should stop *using* `/latest`.
Whether we want to actually remove the URL is an open question, but it's probably good to discuss it elsewhere